app/src/tasks/TaskDetail/Metrics.tsx

```diff
@@ -0,0 +1,95 @@
+import { Await, useLoaderData } from "react-router-dom";
+import Col from "react-bootstrap/Col";
+import { Suspense } from "react";
+import { Task } from "../../ApiClient";
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+import { DateTime } from "luxon";
+import Placeholder from "react-bootstrap/Placeholder";
+import { OutLink } from "../../util";
+
+function FailedMetric({ name, counter }: { name: string; counter: number }) {
+  if (counter > 0) {
+    return (
+      <ListGroup.Item>
+        {name}: {counter}
+      </ListGroup.Item>
+    );
+  } else {
+    return null;
+  }
+}
+
+export default function Metrics() {
+  const { task } = useLoaderData() as {
+    task: Promise<Task>;
+  };
+
+  return (
+    <Col md="6">
+      <Card className="my-3">
+        <Card.Body>
+          <Card.Title>Upload Metrics</Card.Title>
+          <Card.Subtitle>
+            <OutLink href="https://docs.divviup.org/product-documentation/operational-metrics#upload-metrics">
```

Comment:
Before merging I will create this documentation.

Reply:
This is done, right?